python/graphstorm/eval/evaluator.py

```diff
@@ -379,9 +380,32 @@ def do_early_stop(self, val_score):
         else:
             return False
 
+        if early_stop and get_rank() == 0:
+            if self._early_stop_strategy == EARLY_STOP_AVERAGE_INCREASE_STRATEGY:
+                avg_val_perf = sum(self._val_perf_list)/len(self._val_perf_list)
+            elif self._early_stop_strategy == EARLY_STOP_CONSECUTIVE_INCREASE_STRATEGY:
+                val_perf_list = self._val_perf_list.copy()
+
         self._val_perf_list.pop(0)
         self._val_perf_list.append(val_score)
 
+        if early_stop and get_rank() == 0:
+            if self._early_stop_strategy == EARLY_STOP_AVERAGE_INCREASE_STRATEGY:
+                logging.info(
+                    "Early stopping: Current validation score %.3f is lower than "
```

Comment:
Because the evaluation of val values depends on specific metrics, some time we hope to stop earlier if the current val value does not increase, e.g., accuracy and auc, while some time if the val values does not decrease, e.g., mae and rmse. So here we might want to use `validation score %.3f is worse than ...`, rather than saying `lower`, `higher`, `increasing` or `decreasing`.